style.css

```diff
@@ -1,44 +1,45 @@
 @import url("https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap");
```

Comment:
do we need this file?

Reply:
ya to provide font style as suggested in excel sheet